src/MSBuild.UnitTests/CommandLineSwitches_Tests.cs

```diff
@@ -731,6 +731,31 @@ public void PreprocessSwitchIdentificationTests()
             Assert.True(unquoteParameters);
         }
 
+        [Fact]
```

Comment:
Perhaps add a test for failure?

Reply:
added one for multiple occurrences, switch parser does not validate filenames, it accepts anything (like `"\0"`, for example)